### PR TITLE
docs:  fix link in getting started docs

### PR DESCRIPTION
**Pull request by kpcyrd**

### CODE REVIEW

File: docs/_getting-started/index.md [hunk 1]
```diff
@@ -449,4 +449,4 @@ following this with the [Geth with Clef](/docs/getting-started/geth_with_clef) t
 adopt more secure account management practices than those outlined here.
 
 
-[cli]: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line
```
Comment (rjl493456442) [inline docs/_getting-started/index.md:452]: The URL is same. Can you please revert this line of change?

File: docs/_getting-started/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The tutorial on this page assumes Geth and the associated developer tools have b
 
 This page provides step-by-step instructions covering the fundamentals of using Geth. This includes 
 generating accounts, joining an Ethereum network, syncing the blockchain and sending ether between accounts. 
-It is considered best-practice to use [Clef](/docs/clef/Introduction) for account management - this
```
Comment (rjl493456442) [inline docs/_getting-started/index.md:16]: /docs/_clef/Introduction ?

Comment (holiman) [reply docs/_getting-started/index.md:16]: The change in this PR is correct

File: docs/_getting-started/index.md (hunk 1 above)
Comment (holiman) [reply docs/_getting-started/index.md:452]: IIUC it just adds a linebreak

**Review by holiman — APPROVED** [review]
lgtm

**Pull request merged**